src/dev/storybook/aliases.ts

```diff
@@ -55,6 +55,7 @@ export const storybookAliases = {
     'x-pack/plugins/observability_solution/observability_ai_assistant/.storybook',
   observability_ai_assistant_app:
     'x-pack/plugins/observability_solution/observability_ai_assistant_app/.storybook',
+  observability_shared: 'x-pack/plugins/observability_solution/observability_shared/.storybook',
```

Comment:
This will need to be added to `.buildkite/scripts/steps/storybooks/build_and_upload.ts` if you want to preview it in the CI build

Reply:
Perfect, I’ll add it. Thank you very much!